lib/commands/test-init.ts

```diff
@@ -37,6 +37,25 @@ class TestInitCommand implements ICommand {
 				'save-dev': true,
 				optional: false,
 			});
+
+			let modulePath = path.join(projectDir, "node_modules", mod);
+			let modulePackageJsonPath = path.join(modulePath, "package.json");
+			let modulePackageJsonContent = this.$fs.readJson(modulePackageJsonPath);
+			let modulePeerDependencies = modulePackageJsonContent.peerDependencies || {};
+
+			for (let peerDependency in modulePeerDependencies) {
+				let dependencyVersion = modulePeerDependencies[peerDependency] || "*";
+
+				// catch errors when a peerDependency is already installed
+				// e.g karma is installed; karma-jasmine depends on karma and will try to install it again
+				try {
+					await this.$npm.install(`${peerDependency}@${dependencyVersion}`, projectDir, {
+						'save-dev': true
```

Comment:
I don't understand your point. Testing frameworks should be installed as devDependencies so they are not included in built projects, but so that they may still be installed subsequently, for example after removing node_modules.